packages/core/src/base-axis-chart.ts

```diff
@@ -362,6 +383,23 @@ export class BaseAxisChart extends BaseChart {
 				.attr("stroke-width", Configuration.scales.domain.strokeWidth);
 		}
 
+		const tickHeight = this.getLargestTickHeight(yAxisRef.selectAll(".tick"));
+
+		const yAxisTitleTranslate = {
+			x: - (yAxisRef.node().getBBox().height / 2),
+			y: - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5
+		};
+
+		// Add y-axis title
+		if (this.innerWrap.select(".axis-label.y").nodes().length === 0 && this.options.scales.y.title) {
```

Comment:
wouldn't `this.innerWrap.select(".axis-label.y").nodes().length` always be equal to zero here?